docs/settings.md

```diff
@@ -28,12 +28,21 @@ For example, in case you want to run the app on port `5000`, just set the enviro
 
 ## Development
 
-* `--reload` - Enable auto-reload.
+* `--reload` - Enable auto-reload. Uvicorn supports two versions of auto-reloading behavior enabled by this option. There are important differences between them.
 * `--reload-dir <path>` - Specify which directories to watch for python file changes. May be used multiple times. If unused, then by default the whole current directory will be watched. If you are running programmatically use `reload_dirs=[]` and pass a list of strings.
+
+### Naive Reloading
+
+If Uvicorn _cannot_ load [watchgod](https://pypi.org/project/watchgod/) at runtime, it will periodically look for changes in modification times to all `*.py` files (and only `*.py` files) inside of its monitored directories. See the `--reload-dir` option. Which files are monitored is not configurable unless watchgod in installed. See the `--reload-include` and `--reload-exclude` options for details.
```

Comment:
@Andrew-Chen-Wang, here's my proposal which is hopefully responsive to [your recent observations](https://github.com/encode/uvicorn/pull/1331#discussion_r786256431) about clarity regarding which `*.py` files would trigger reloads in this context.

Reply:
Just misspelling (is vs. in installed) and clarity. Otherwise, LGTM!:

```suggestion
If Uvicorn _cannot_ load [watchgod](https://pypi.org/project/watchgod/) at runtime, it will periodically look for changes in modification times to all `*.py` files (and only `*.py` files) inside of its monitored directories. See the `--reload-dir` option. Specifying other file extensions is not supported unless watchgod is installed. See the `--reload-include` and `--reload-exclude` options for details.
```

Reply:
Awesome catch! Thanks!